### DISCUSSION TITLE
My computer can't access Github unless I'm using VPN

### DISCUSSION THREAD

**Discussion by Telemocus** [community/community#53718] [New to GitHub]
My computer can't access Github unless I'm using VPN. Here is the error code. After running Git debug

C:\Windows\System32>git clone git@github.com:github/debug-repo debug-repo-ssh
Cloning into 'debug-repo-ssh'...
ssh: Could not resolve hostname github.com: No address associated with hostname
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

C:\Windows\System32>ping -n 10 github.com
Ping request could not find host github.com. Please check the name and try again.

C:\Windows\System32>tracert github.com
Unable to resolve target system name github.com.

**Comment by rxerium** [comment]
Could possibly be a DNS issue, are you able to ping google.com or Twitter.com?

**Comment by Telemocus** [comment]
On Tue, 25 Apr 2023 at 21:46, Rishi ***@***.***> wrote:

> Could possibly be a DNS issue, are you able to ping google.com or
> Twitter.com?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/53718#discussioncomment-5724035>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2UQVDIPYJ77FTMI6DAWZW3XDASYHANCNFSM6AAAAAAXLN6OOY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
Yes I am able to ping both Twitter.com and  Google.com. Thought it was a
dns issue and tried flushing it but nothing

**Comment by iamreiyn** [comment]
Hi @Telemocus 

Try changing your DNS server to a different one, such as Google DNS (8.8.8.8 and 8.8.4.4) or Cloudflare DNS (1.1.1.1 and 1.0.0.1). This can be done in your network settings. Feel free to let us know if the problem still persists.
